examples/jsm/loaders/LDrawLoader.d.ts

```diff
@@ -0,0 +1,21 @@
+import {
+	LoadingManager,
+	Group,
+	Material
+} from '../../../src/Three';
+
+export class LDrawLoader {
+	constructor(manager?: LoadingManager);
+	manager: LoadingManager;
+	path: string;
+
+	load(url: string, onLoad: (data: Group) => void, onProgress?: (event: ProgressEvent) => void, onError?: (event: ErrorEvent) => void) : void;
+	setPath(path: string) : this;
+	setFileMap(fileMap: Record<string, string>): void;
```

Comment:
Is it ok to use `Record` to express a hash object here? (string -> string)

Reply:
I've not used `Record` so far but I think it's appropriate for this context.